app/uk/gov/hmrc/platforminitiatives/services/PlatformInitiativesService.scala

```diff
@@ -167,4 +179,38 @@ class PlatformInitiativesService @Inject()(
         )
       }
     }
+
+  def createMigrationInitiative(
+   initiativeName              : String,
+   initiativeDescription       : String,
+   newGroup                    : String,
+   newArtefact                 : String,
+   oldGroup                    : String,
+   oldArtefact                 : String,
+   team                        : Option[String] = None,
+   environment                 : Option[Environment] = Some(Environment.Production),
+   completedLegend             : String = "Completed",
+   inProgressLegend            : String = "Not Completed",
+   experimental                : Boolean = false
+ )(implicit ec: ExecutionContext): Future[PlatformInitiative] = {
+    for {
+      firstArtefactDependencies   <- serviceDependenciesConnector.getServiceDependency(newGroup, newArtefact, environment)
+      secondArtefactDependencies  <- serviceDependenciesConnector.getServiceDependency(oldGroup, oldArtefact, environment)
+      allDependencies             = firstArtefactDependencies ++ secondArtefactDependencies
+    } yield PlatformInitiative(
+      initiativeName              = initiativeName,
+      initiativeDescription       = initiativeDescription,
+      progress                    = Progress(
+        current = allDependencies
+          // Filtering for exclusively owned repos
```

Comment:
Remind me why we do this again? I think there are quite a few repos that have two owners - e.g. the original team, and then live services

Reply:
This is for the `/platform-initiatives/teams/{TEAM_NAME}` endpoints. If None, returns all repos.